int.md

```diff
@@ -23,41 +23,72 @@ In CockroachDB, the following are aliases for `INT`:
 ## Syntax
 
 A constant value of type `INT` can be entered as a [numeric literal](sql-constants.html#numeric-literals).
-For example: `42`, `-1234` or `0xCAFE`.
+For example: `42`, `-1234`, or `0xCAFE`.
 
-## Size
+## Length
+
+CockroachDB does not offer multiple integer types for different lengths; instead, our compression ensures that smaller integers use less disk space than larger integers.
+
+However, you can use `BIT(n)` to limit integers based on their corresponding binary values. For example, `BIT(5)` would allow `31` because it corresponds to the five-digit binary integer `11111`, but would not allow `32` because it corresponds to the six-digit binary integer `100000`, which is 1 bit too long. See the [example](#examples) below for a demonstration.
 
-An `INT` column supports values up to 8 bytes in width, but the total storage size is likely to be larger due to CockroachDB metadata. 
+{{site.data.alerts.callout_info}}Although a column defined as <code>BIT(n)</code> is represented as such in <a href="show-columns.html"><code>SHOW COLUMNS</code></a> and other modes of introspection, values stored in the column are decimal integers, not binary integers. Also note that <code>BIT</code> is equivalent to <code>BIT(1)</code>.{{site.data.alerts.end}}
```

Comment:
All integers are *stored* as binary values, but *displayed* as decimal values. I'd phrase this something like "The use of the `BIT` type does not imply that values will be displayed (or input) in a binary format; `BIT` values are displayed and input by default in decimal format like all other integers".